nightshift/marc/marc_writer.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides methods for manipulation and serialization of Worldcat responses
+into MARC21.
+"""
+import logging
+import pickle
+
+from pymarc import Field
+
+from .. import __title__, __version__
+from ..constants import (
+    library_by_id,
+    resource_category_by_id,
+    sierra_format_code,
+    tags2delete,
+)
+from ..datastore import Resource
+from .marc_parser import worldcat_response_to_pymarc
+
+
+logger = logging.getLogger("nightshift")
+
+
+DELETE_TAGS = tags2delete()
+LIB_IDX = library_by_id()
+RES_IDX = resource_category_by_id()
+SIERRA_FORMAT = sierra_format_code()
+
+
+class BibEnhancer:
+    """
+    A class used for upgrading MARC records.
+    """
```

Comment:
It would be great if this docstring could have a little more detail, or if the overall work being done by this class was explained somewhere. I can see the shape of changes being made by the `manipulate` method but I think a little more detail would be useful!

Reply:
Good thinking. I hope this provides enough detail: [#8a42627](https://github.com/BookOps-CAT/NightShift/pull/11/commits/8a426276ec490ce8195a090c0e88ab3b686dfe35)